hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsStreamStatistics.java

```diff
@@ -76,12 +79,8 @@ public void testAbfsStreamOps() throws Exception {
       assertReadWriteOps("read", 1, statistics.getReadOps());
 
     } finally {
-      if (inForOneOperation != null) {
-        inForOneOperation.close();
-      }
-      if (outForOneOperation != null) {
-        outForOneOperation.close();
-      }
+      IOUtils.cleanupWithLogger(null, inForOneOperation,
```

Comment:
you are going to need to create a logger in this test case and pass it down i'm afraid